mattermost-plugin/webapp/src/index.tsx

```diff
@@ -236,8 +236,8 @@ export default class Plugin {
         } else {
             windowAny.frontendBaseURL = subpath + '/plug/focalboard'
             this.channelHeaderButtonId = registry.registerChannelHeaderButtonAction(<FocalboardIcon/>, () => {
-                const currentChannel = mmStore.getState().entities.channels.currentChannelId
-                window.open(`${window.location.origin}/plug/focalboard/workspace/${currentChannel}`)
+                const currentTeam = mmStore.getState().entities.teams.currentTeamId
+                window.open(`${window.location.origin}/plug/focalboard/team/${currentTeam}`)
```

Comment:
Any reason not doing the `_blank` and `noopener` like above here too?

```suggestion
                window.open(`${window.location.origin}/plug/focalboard/team/${currentTeam}`, '_blank', 'noopener')
```

Reply:
After spending some time investigating, I'm not sure the "else" section even gets hit. The path provided `/plug/focalboard` is wrong as well, leading me to believe the section is unnecessary.